pinot-core/src/test/java/org/apache/pinot/core/common/DataFetcherTest.java

```diff
@@ -363,7 +361,8 @@ public void testFetchBytesValues(String column) {
     _dataFetcher.fetchBytesValues(column, docIds, length, bytesValues);
 
     for (int i = 0; i < length; i++) {
-      Assert.assertEquals(new String(bytesValues[i], UTF_8), Integer.toString(_values[docIds[i]]), ERROR_MESSAGE);
+      String stringValue = Integer.toString(_values[docIds[i]]);
+        Assert.assertEquals(new String(bytesValues[i], UTF_8), stringValue, ERROR_MESSAGE);
```

Comment:
(format) reformat, or revert since it doesn't change the original logic

Reply:
Done